packages/fast-colors/src/color-quantization.ts

```diff
@@ -0,0 +1,197 @@
+// tslint:disable:member-ordering
+// tslint:disable:no-bitwise
+// tslint:disable:prefer-for-of
+
+import { PixelBlob } from "./pixel-blob";
+import { Histogram } from "./histogram";
+import { insertIntoSortedList, PixelBox } from "./pixel-box";
+import { ColorRGBA64 } from "./color-rgba-64";
+
+export interface QuantizedColor {
+    color: ColorRGBA64;
+    pixelCount: number;
+    colorVolume: number;
+}
+
+export interface QuantizeConfig {
+    // must be in the range [1,8]. Memory use increases as 4*2^(3*signifigantBits).
```

Comment:
I imagine this info in comments would be usefull for app authors - if you use JSDOC comments the comment will survive the typescript compile process and can be made availible to editors like VSCode.

```
/*
 * my comment
 */
```

It would be awesome if you could update the comments for publicly exported objects to be the above syntax